examples/auto-deploy/velero.sh

```diff
@@ -0,0 +1,233 @@
+#!/bin/bash
+
+ACCESS_KEY=${ACCESS_KEY:-"konkhmer"}
+SECRET_KEY=${SECRET_KEY:-"konkhmer855"}
```

Comment:
Are these default values your user names? I think something more generic would be appropriate here.

Reply:
Okay, those values has been changed.